submission/submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/submitter/SubmissionSubmitter.kt

```diff
@@ -65,16 +65,26 @@ class SubmissionSubmitter(
         try {
             logger.info { "${rqt.accNo} ${rqt.owner} Started processing submission request" }
 
-            rqt.draftKey?.let { draftService.setProcessingStatus(rqt.owner, it) }
+            rqt.draftKey?.let {
```

Comment:
I think this is adding too much complexity to this method, we times the same conditional (if draftkey != null)  in this few lines.  Not really sure about but we may consider include accNo and owner in `draftService` operations to log on each status change.  Not much better as we will be including parameters only for logging purposes but as it is righ it looks super verbose.